torchvision/transforms/functional.py

```diff
@@ -153,7 +161,7 @@ def convert_image_dtype(image: torch.Tensor, dtype: torch.dtype = torch.float) -
         return image.mul(torch.iinfo(dtype).max + 1 - eps).to(dtype)
     else:
         # int to float
-        if dtype.is_floating_point:
+        if output_is_float:
             max = torch.iinfo(image.dtype).max
```

Comment:
iinfo isn't scriptable either
https://github.com/pytorch/pytorch/issues/41492

Reply:
Correct. That is why we decided in https://github.com/pytorch/vision/pull/2313#issuecomment-643177380 to put this on hold until `torch.iinfo(dtype)` and `torch.dtype.is_floating_point` are supported by `torch.script`.